src/routes/exchange/exchange.ts

```diff
@@ -29,6 +29,7 @@ export class Exchange {
     private environment = environment;
     private controller: ValidationController;
     private renderer: BootstrapFormRenderer;
+    @observable({ changeHandler: 'currentTokenChanged' })
```

Comment:
This is great. Aurelia will by default using convention add Changed to the end of your observed value, so you don't have to add the change handler here. Not a biggie :)